app/src/features/projects/__snapshots__/CreateProjectPage.test.tsx.snap

```diff
@@ -502,16 +5039,43 @@ exports[`CreateProjectPage renders the initial default page correctly 1`] = `
         >
           Create Project
         </h1>
+        <button
+          class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-actionButton-1 MuiButton-outlinedPrimary"
+          tabindex="0"
+          type="button"
+        >
+          <span
+            class="MuiButton-label"
+          >
+            Save as Draft
+          </span>
+          <span
+            class="MuiTouchRipple-root"
+          />
+        </button>
       </div>
       <div
         class="MuiBox-root MuiBox-root-10"
+      >
+        <div
+          class="MuiBox-root MuiBox-root-11"
+        >
+          <span
+            class="MuiTypography-root MuiTypography-subtitle2 MuiTypography-colorTextSecondary"
+          >
+            Draft saved on 
```

Comment:
question: there is no date here... should there be?

Reply:
Yeah its fine.  I render the string always, and just selectively make it visible.
Only reason is so the page doesnt get pushed down when the message shows up.  Probably a fancier way to do it, but I didnt want to get into it when this was a bonus thing I was adding not in the design.